src/db/mysql/schema.rs

```diff
@@ -10,11 +12,14 @@ table! {
 
 table! {
     bso (user_id, collection_id, id) {
+        #[sql_name="userid"]
         user_id -> Integer,
+        #[sql_name="collection"]
         collection_id -> Integer,
         id -> Varchar,
         sortindex -> Nullable<Integer>,
         payload -> Mediumtext,
+        #[sql_name="modified"]
         modified -> Bigint,
         expiry -> Bigint,
```

Comment:
`expiry` is `ttl` in the old schema :(

also collection's `id` column is `collectionid`

and of course, mysql's batch schema/implementation is completely different, which is really a whole other issue entirely (should we match its schema + old impl.?)

Reply:
hmm. I was using `server-syncstorage` repo for the schema. Is there a better source I can use to verify against?

Reply:
Yup, I'm looking at its sqlalchemy definitions:

https://github.com/mozilla-services/server-syncstorage/blob/5e9045b/syncstorage/storage/sql/dbconnect.py#L132

Reply:
Ah, cool. I just missed that one. Triple checking now. Thanks!